pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/util/ConfigManager.java

```diff
@@ -121,7 +117,7 @@ public Config getConfig() {
         for (Map.Entry<String, String> override : overrides.entrySet()) {
             ConfigValue overrideValue = ConfigValueFactory.fromAnyRef(override.getValue(), "Overriding via " + this.getClass().getName());
             cfgWithOverride = cfgWithOverride.withValue(override.getKey(), overrideValue);
-            LOG.info("Overriding config value {} with {}", override.getKey(),  override.getValue());
+            log.info("Overriding config value {} with {}", override.getKey(),  override.getValue());
```

Comment:
Let's remove logging of the value.

```suggestion
            log.info("Overriding config value for key {}", override.getKey());
```